tests/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_C_COMPILER "$ENV{HOME}/cross/bin/aarch64-w64-mingw32-gcc")
```

Comment:
I'd rather change this to `$ENV{INSTALL_PATH}/bin/${TARGET}-gcc` to be more compatible with `.github/scripts/build.sh`. The default of `INSTALL_PATH` can be `$ENV{HOME}/cross`.